source/fundamentals/crud/read-write-pref.txt

```diff
@@ -0,0 +1,235 @@
+.. _node-crud-write-read-pref:
+
+===============================================
+Specify How CRUD Operations Run on Replica Sets
+===============================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta:: 
+   :keywords: node.js, customize, preferences, replica set, consistency
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the **write concern**, **read concern**, and
+**read preference** configurations to modify the way that MongoDB runs
+create, read, update, and delete (CRUD) operations on replica sets.
+
+You can set write concern, read concern, and read preference options at the following
+levels:
+
+- Client, which sets the *default for all operation executions* unless overridden
+- Session
+- Transaction
+- Database
+- Collection
+
+These options allow you to customize the consistency and availability of the data in your
+replica sets.
+
+Write Concern
+-------------
+
+The write concern specifies the level of acknowledgement requested from MongoDB for write
+operations, such as an insert or update, before the operation successfully returns.
+Operations that do not specify an explicit write concern inherit the global default write
+concern settings.
+
+For more information, see :manual:`Write Concern </reference/write-concern/>` in the
+Server manual. For more detailed API descriptions, see the `WriteConcern API documentation
+<{+api+}/classes/WriteConcern.html>`__.
+
+A ``WriteConcern`` accepts the following parameters:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Parameter
+     - Description
+  
+   * - ``w`` *(optional)*
+     - | Requests acknowledgment that the write operation has propagated to a specified number of mongod instances or to mongod instances with specified tags 
+       | **Type:** `W <{+api+}/types/W.html>`__
+
+   *  - ``wtimeoutMS`` *(optional)*
+      - | Specifies a time limit to prevent write operations from blocking indefinitely 
+        | **Type:** number
+
+   * - ``journal`` *(optional)*
+     - | Requests acknowledgment that the write operation has been written to the on-disk journal 
+       | **Type:** boolean
+
+Example: Set the Write Concern for a Single Write Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This code uses custom ``WriteConcern`` settings while creating new a document:
+
+.. code-block:: js
+
+   myDB.myCollection.insertOne( 
+      { name: "anotherDocumentName" }, 
+      { writeConcern: 
+        { w: 2, wtimeoutMS: 5000 } 
+      } 
+   );
+
+Example: Create and Use a New Write Concern using the fromOptions and apply Methods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   
+This code use the ``fromOptions`` method to construct a ``WriteConcern`` from the options of an
```

Comment:
Suggestion:

Method notation from a [prior suggestion](https://github.com/mongodb/docs-node/pull/819#discussion_r1401153101) and a grammar fix:

```suggestion
This code uses the ``fromOptions()`` method to construct a ``WriteConcern`` from the options of an
```